CMakeLists.txt

```diff
@@ -1,20 +1,43 @@
 cmake_minimum_required(VERSION 3.18)
 project(file-editor.out)
 
+option(ENABLE_DOCTESTS "Include tests in the library. Setting this to OFF will remove all doctest related code.
+                        Tests in tests/*.cpp will still be enabled." ON)
+
+set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_SOURCE_DIR}/cmake/")
+include(ConfigSafeGuards)
+include(Doctest)
+
 set(CMAKE_CXX_STANDARD 17)
-set(CMAKE_CXX_FLAGS_RELEASE "${CMAKE_CXX_FLAGS_RELEASE} -DDOCTEST_CONFIG_DISABLE")
-set(EXECUTABLE_OUTPUT_PATH bin/)
+set(EXECUTABLE_OUTPUT_PATH ../bin/)
```

Comment:
I don't think that it is a good idea to have the executable in the `../bin` folder when every other build artifact is in the `build` folder, e.g. the `unit_tests`.

Reply:
All will now be built inside the build folder!